src/hotspot/share/opto/superword.cpp

```diff
@@ -2359,472 +2052,6 @@ void SuperWord::schedule_reorder_memops(Node_List &memops_schedule) {
   }
 }
 
-//------------------------------output---------------------------
-// Convert packs into vector node operations
-// At this point, all correctness and profitability checks have passed.
-// We start the irreversible process of editing the C2 graph. Should
-// there be an unexpected situation (assert fails), then we can only
-// bail out of the compilation, as the graph has already been partially
-// modified. We bail out, and retry without SuperWord.
-bool SuperWord::output() {
-  CountedLoopNode *cl = lpt()->_head->as_CountedLoop();
-  assert(cl->is_main_loop(), "SLP should only work on main loops");
-  Compile* C = phase()->C;
-  if (_packset.is_empty()) {
-    return false;
-  }
-
-#ifndef PRODUCT
-  if (TraceLoopOpts) {
-    tty->print("SuperWord::output    ");
-    lpt()->dump_head();
-  }
-#endif
-  phase()->C->print_method(PHASE_SUPERWORD2_BEFORE_OUTPUT, 4, cl);
-
-  adjust_pre_loop_limit_to_align_main_loop_vectors();
-
-  DEBUG_ONLY(verify_no_extract());
```

Comment:
Note: now call `adjust_pre_loop_limit_to_align_main_loop_vectors` from `VTransformGraph::apply`. And `verify_no_extract` is directly called in `SuperWord::SLP_extract`.